modules/is_analytics_dashboard_master/manifests/init.pp

```diff
@@ -30,7 +30,7 @@
   file { "binary":
     path   => "${distribution_path}/${product_binary}",
     mode   => '0644',
-    source => "puppet:///modules/distributions/${product_binary}",
+    source => "puppet:///modules/commons/${product_binary}",
```

Comment:
Shall we rename this to `common` to keep the consistency with the ansible roles[1]? IMO `common` is more appropriate than `commons`.
1. https://github.com/wso2/ansible-apim/tree/master/roles/common